wled00/e131.cpp

```diff
@@ -34,7 +34,7 @@ void handleDDPPacket(e131_packet_t* p) {
 
   realtimeLock(realtimeTimeoutMs, REALTIME_MODE_DDP);
   
-  if (!realtimeOverride) {
+  if (!realtimeOverride || (realtimeMode && useMainSegmentOnly)) {
```

Comment:
I think override should override Live mode if on main seg only too. Otherwise override never functions if set to main segment only.

Reply:
There is no "real" need for override in mainseg only if you can cancel live. You can argue that a user may want displaying effects instead of live data on mainseg while not cancelling live streaming (temporary).
This will need a new UI element (perhaps a star in the segment?)

Reply:
This just gave me an idea!